pulse/pulse/views.py

```diff
@@ -12,22 +12,27 @@ def register(app):
     def index():
         return render_template("en/index.html")
 
-    # English routes
-    @app.route("/en/organizations/")
-    def index_en():
-        return render_template("en/index.html")
+    @app.route("/<prefix>/organizations/")
+    def organizations(prefix):
+        return render_template(generate_path(prefix, "index"))
+
+    @app.route("/<prefix>/domains/")
+    def https_domains(prefix):
+        return render_template(generate_path(prefix, "domains"))
 
-    @app.route("/en/domains/")
-    def https_domains():
-        return render_template("en/domains.html")
+    @app.route("/<prefix>/guidance/")
+    def guidance(prefix):
+        return render_template(generate_path(prefix, "guidance"))
 
-    @app.route("/en/guidance/")
-    def guidance():
-        return render_template("en/guidance.html")
+    @app.route("/<prefix>/feedback/")
+    def feedback(prefix):
+        return render_template(generate_path(prefix, "feedback"))
 
-    @app.route("/en/feedback/")
-    def feedback():
-        return render_template("en/feedback.html")
+    def generate_path(prefix, page_id):
+        if(prefix == 'en' or prefix == 'fr'):
+            return prefix + "/" + page_id + ".html"
```

Comment:
There are many options with string interpolation in python, but concatenating with `+` is generally avoided.

I would suggest doing `f'{prefix}/{page_id}.html'` instead.